packages/flutter/lib/src/rendering/viewport.dart

```diff
@@ -1188,6 +1209,11 @@ class RenderViewport extends RenderViewportBase<SliverPhysicalContainerParentDat
   /// The semantic scrolling actions and the [SemanticsNode]s of scrollable
   /// children will be attached to the inner node, which itself is a child of
   /// the outer node.
+  ///
+  /// See also:
+  ///
+  /// * [RenderViewportBase.describeSemanticsConfiguration], which adds this
+  ///   tag to its [SemanticsConfiguration] as described above.
```

Comment:
nit, let's remove the `as described above.` It seems to not add much information